exampleCourse/courseInstances/Sp15/assessments/exam05-submittableAccessRestriction/infoAssessment.json

```diff
@@ -0,0 +1,44 @@
+{
+    "uuid": "75be4222-e9d1-11eb-9a03-0242ac130003",
+    "type": "Exam",
+    "title": "Submittable Access Rule Demo",
+    "set": "Exam",
+    "number": "5",
+    "allowAccess": [
+        {
+            "role": "TA",
+            "credit": 100
+        },
+        {
+            "mode": "Public",
+            "credit": 100,
+            "timeLimitMin": 50,
+            "startDate": "2000-01-02T00:00:01",
+            "endDate": "2000-01-02T23:59:59",
+            "showClosedAssessment": false
+        },
+        {
+            "mode": "Public",
+            "credit": 100,
+            "timeLimitMin": 50,
+            "startDate": "2100-01-02T00:00:01",
+            "endDate": "2100-01-02T23:59:59",
+            "showClosedAssessment": false
+        },
+        {
+            "mode": "Public",
+            "submittable": false,
+            "showClosedAssessment": false
```

Comment:
I think we don't want `"showClosedAssessment": false` here, because it will block students from viewing their closed assessments?

Also, maybe we can add a `"comment": "This access rule is for viewing the exam after it is finished"`?

And we should probably have a `startDate` set so that exam viewing only starts a couple of days after the exam ended?

Reply:
Fixed